data/data/bootstrap/files/usr/local/bin/keepalived.sh

```diff
@@ -35,7 +35,7 @@ for iface_cidr in iface_cidrs:
 sys.exit(1)
 EOF
 )
-    ip -o addr show to "$net_cidr" | awk '{print $2}'
+    ip -o addr show to "$net_cidr" | head -n 1 | awk '{print $2}'
```

Comment:
I would really prefer if we have some stricter logic than just choosing whichever address was configured first for the link. It should be fine to merge this in the meantime.